.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!-- Thanks for opening a pull request! Please do not just delete this text. -->
+
+#### Summary
+<!--
+ Explain the **motivation** for making this change. What existing problem does the pull request solve? How can reviewers test this PR?
+
+Please remember to:
+- Pair the PR with an issue. If it is a new feature, the issue should proceed the PR and will have allowed sufficent time for discussions to take place. Pleases use
+issue tags such as "Closes #XYZ" or "Resolves sigstore/repo-name#XYZ".
+  [documentation](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
+- ensure your commits are signed-off, as sigstore uses the [DCO](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin) using `git commit -s`, or `git commit -s --amend` if you want to amend already existing commits
+
+Thank you :)
+-->
+
+##### Checklist
```

Comment:
optional: was having this indented one level below summary intentional?

Reply:
Yes, wanted to show just as a bolded header, but we don't have a separate section from Summary